pom.xml

```diff
@@ -138,13 +137,13 @@
     <maven-dependency.version>3.1.1</maven-dependency.version>
     <maven-versions.version>2.8.1</maven-versions.version>
     <commons-io.version>2.1</commons-io.version>
-
     <!-- Dependency properties -->
     <commons-codec.version>1.11</commons-codec.version>
     <eddsa.version>0.3.0</eddsa.version>
     <grpc.version>1.33.0</grpc.version>
     <jackson-databind.version>2.9.10.7</jackson-databind.version>
-    <javax.annotation-api.version>1.3.2</javax.annotation-api.version>
+    <javax.annotation-api.version>
```

Comment:
nit: I believe due to code alignment these lines moved to next line. These changes can be reverted for readability.